ui/app/rif/classes/token.js

```diff
@@ -0,0 +1,12 @@
+export default class Token {
+  constructor (name, symbol, address, networkChannels, userChannels, networkAddress, joined, userBalance) {
+    this.name = name;
+    this.symbol = symbol;
+    this.address = address;
+    this.networkChannels = networkChannels;
+    this.userChannels = userChannels;
+    this.network_address = networkAddress;
```

Comment:
lets use cammel case here

Reply:
Will fix this in next commit, nice catch here!